### PR TITLE
feat: Add tid to avatar

### PR DESCRIPTION
**Pull request by Vibes-INS**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `develop` branch(or to a previous version branch), _not_ the `master` branch
- [ ] All tests are passing
- [ ] Build success

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature

**Other information:**

### CODE REVIEW

**Review by ybian19 — APPROVED** [review]
verified

File: src/views/Profile/ImagePreview.tsx [hunk 1]
```diff
@@ -88,10 +88,10 @@ export const ImagePreview: React.FC = () => {
 
   const [datauri, ext, fromCamera, tokenUuid] = useMemo(() => {
     const locale = i18n.language === 'en' ? 'en' : 'zh'
-    let datauri = location?.state?.datauri ?? ''
-    datauri = location.state.tid
-      ? addLocaleToUrl(addTidToUrl(datauri, location.state.tid), locale)
-      : datauri
+    const datauri = addParamsToUrl(location?.state?.datauri ?? '', {
+      locale,
+      ...(location.state.tid ? { tid: location.state.tid } : {}),
```
Comment (yuche) [inline src/views/Profile/ImagePreview.tsx:93]: location 可能没有 state

File: src/views/NFT/index.tsx [hunk 2]
```diff
@@ -358,12 +358,12 @@ export const NFT: React.FC = () => {
 
   const { renderer, bgImgUrl } = useMemo(() => {
     const nftDetail = detail as NFTDetail
-    const isClass = nftDetail?.n_token_id !== undefined
+    const isNft = nftDetail?.n_token_id !== undefined
```
Comment (yuche) [inline src/views/NFT/index.tsx:361]: 这里是不是没必要重命名，没有 n_token_id 他就是个 class token

**Review by yuche — APPROVED** [review]

File: src/views/NFT/index.tsx (hunk 2 above)
Comment (Vibes-INS) [reply src/views/NFT/index.tsx:361]: 是哦

**Pull request merged**